app/experimenter/kinto/tasks.py

```diff
@@ -213,3 +213,36 @@ def nimbus_check_experiments_are_complete():
                 logger.info(f"{experiment} status is set to Complete")
 
     metrics.incr("check_experiments_are_complete.completed")
+
+
+@app.task
+@metrics.timer_decorator("end_experiment_in_kinto")
+def nimbus_end_experiment_in_kinto(experiment_id):
```

Comment:
So if we set the task up like this, as a single invocation, and it fails (the common case of which will be that the collection already has a pending change), then it'll never retry.  So I think a more hardened approach is to

1) mutation sets `is_end_requested` bool to True
2) a scheduled task wakes up and checks for experiments that have end requested but are still present in the collection
3) if the collection is not currently pending a review, then we can call `delete_from_kinto`

So since we already have a task that does most of that here:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/kinto/tasks.py#L80

We can probably slot the removal in right here:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/kinto/tasks.py#L126

instead of making this a separate single invocation task.